pkg/controller/statefulset/stateful_pod_control_test.go

```diff
@@ -124,7 +124,7 @@ func TestStatefulPodControlCreatePodPvcCreateFailure(t *testing.T) {
 	}
 	for i := range events {
 		if !strings.Contains(events[i], v1.EventTypeWarning) {
-			t.Errorf("Expected normal events found %s", events[i])
+			t.Errorf("Expected normal event found %s", events[i])
```

Comment:
The original message seems inaccurate. According to the code it should have said `"found unexpected non-warning event: %s"`